### ISSUE TITLE
max character count 

### ISSUE THREAD

**Issue by breadguyyy** [kokonect-link/cherrypick#539]
### Summary

increase max character count to 100k like other forks/instances

### Purpose

good for federation with those instances and also character limits stinky

### Do you want to implement this feature yourself?

- [ ] Yes, I will implement this by myself and send a pull request

**Comment by Linux-Is-Best** [comment]
2 things:

1) The admin is the one who sets the character limit.

2) 100,000 characters would put a straign on most servers.

On a side not, that would be more character than the first few pages of the New York Times.

**Comment by breadguyyy** [comment]
8192 is the hardcoded max iirc

**Comment by Linux-Is-Best** [comment]
100k is far to much. 

Best to close this and suggest to make it an admin option. Let each admin decide.

**Comment by breadguyyy** [comment]
yeah that's what I'm asking, to raise the hardcoded max character count so admins can choose more than 8192
